recipes/rust-ncbitaxonomy/build.sh

```diff
@@ -11,4 +11,4 @@ fi
 LD_LIBRARY_PATH="$PREFIX/lib"
 export LD_LIBRARY_PATH
 # build statically linked binary with Rust
-RUST_BACKTRACE=1 C_INCLUDE_PATH=$PREFIX/include cargo install --verbose --root $PREFIX --path .
+RUST_BACKTRACE=1 C_INCLUDE_PATH=$PREFIX/include cargo install --no-track --verbose --root $PREFIX --path .
```

Comment:
Without `--no-track` this created some
```
.crates2.json
.crates.toml
```
files.

Reply:
ok thanks, I didn't know that.

Reply:
Me neither, just noticed those surplus files and `--no-track` is supposed to not create them AFAIU.